### ISSUE TITLE
Blob finding runtime warning

### ISSUE THREAD

**Issue by henrypinkard** [scikit-image/scikit-image#4736]
## Description

Running the following code:

```
from skimage.feature import blob_dog
import numpy as np

image = np.load('bad_array.npz')['arr_0']
blob_dog(image, min_sigma=5, max_sigma=50, threshold=0.8, overlap=0.6)
```
gives 

```
/Users/henrypinkard/anaconda3/lib/python3.7/site-packages/skimage/feature/blob.py:125: RuntimeWarning: invalid value encountered in double_scalars
  r1 = blob1[-1] / blob2[-1]
/Users/henrypinkard/anaconda3/lib/python3.7/site-packages/skimage/feature/blob.py:126: RuntimeWarning: divide by zero encountered in true_divide
  pos1 = blob1[:ndim] / (max_sigma * root_ndim)
/Users/henrypinkard/anaconda3/lib/python3.7/site-packages/skimage/feature/blob.py:127: RuntimeWarning: divide by zero encountered in true_divide
  pos2 = blob2[:ndim] / (max_sigma * root_ndim)
/Users/henrypinkard/anaconda3/lib/python3.7/site-packages/skimage/feature/blob.py:129: RuntimeWarning: invalid value encountered in subtract
  d = np.sqrt(np.sum((pos2 - pos1)**2))
```

Doesn't seem to cause problems with the output, but I thought it might be worth looking into

## Version information

3.7.0 (default, Jun 28 2018, 07:39:16) 
[Clang 4.0.1 (tags/RELEASE_401/final)]
Darwin-18.7.0-x86_64-i386-64bit
scikit-image version: 0.17.2
numpy version: 1.18.1

[bad_array.npz.zip](https://github.com/scikit-image/scikit-image/files/4660373/bad_array.npz.zip)

**Comment by sciunto** [comment]
Xref: #4729

**Comment by emmanuelle** [comment]
@henrypinkard could you please check that this is fixed on master (thanks to #4803) and close the issue if yes?

**Comment by henrypinkard** [comment]
Yep, fixed, thanks!
